Updated for proper upload link

fbane opened the pull request:
- Updated to create the saves/ and saves/public/ if doesn't exist and removed the creation of public/
- Modified code to create proper link if save file exists in /saves/public/ and will be black if the upload user did not choose public
- Currently placeholder number in place for the links to show expired; Will make another PR after this to reflect that code change as well.

fbane committed:
Updated for proper upload link

ian76g commented on index.php:
```diff
@@ -102,7 +103,7 @@ function mysort($a, $b)
                             $dl = '';
-                            if (file_exists(SHELL_ROOT . 'public/' . substr($file, 5, -5) . '.sav')) {
+                            if (file_exists(SHELL_ROOT . 'maps/' . substr($file, 5, -5) . '.sav')) {
                                 $dl = ' (DL)';
```
this one is wrong - DL (downLoad) should only be possible for files in /saves/public.
here you are checking (maps

ian76g commented on index.php:
```diff
@@ -130,19 +131,25 @@ function mysort($a, $b)
+                            // Check to see if savefile exists in public folder and create download link
+                            if (file_exists(SHELL_ROOT.'saves/public/' . $saveFile)) {
+                              $upTime = filemtime($saveCheck);
+                              $timeCheck = time() - $upTime;
```
$saveCheck is no longer defined - you removed original line 134

fbane committed:
Updates to index.php

- Reduced table headers to fit better
- Removed and updated variable for time check
